src/components/Footer.js

```diff
@@ -29,23 +31,25 @@ class Footer extends Component{
 							<a href="https://www.facebook.com/groups/XSURGEDEFI" target="_blank">
 								<Image src="assets/img/facebook.png" className="footer-image" id="footer_social_last" />
 							</a>
-						</div>
-						<div id="copyright_links">
-							© 2021 xSurge. All rights reserved.
-						</div>
-						<div class="clear"></div>
-						<div id="disclosure_text">
-							<LinkContainer to="/disclosures" className="disclosure_link">
-								<span id="disclosure_text_span">Disclosures</span>
-							</LinkContainer>
-							<span> | </span>
-							<LinkContainer to="/disclosures" className="disclosure_link">
-								<span id="disclosure_text_span">*Surge Price</span>
-							</LinkContainer>
-						</div>
-					</div>
-				</Col>
-			</Row>
+						</Col>
+						</Row>
+						
+						{/* <div class="clear"></div> */}
+
+						<Row>
+						<Col xs={12} sm={12} md={12} lg={12} xl={12}>
+							<div class="text-center text-md-start mt-5px" id="disclosure_text">
```

Comment:
Explanation of the logic: So it centers on mobile first. But on medium screens, it's right aligned (text-md-start).